cmd/autometrics/main.go

```diff
@@ -24,6 +24,8 @@ type args struct {
 	UseOtel              bool   `arg:"--otel" default:"false" help:"Use OpenTelemetry client library to instrument code instead of default Prometheus."`
 	AllowCustomLatencies bool   `arg:"--custom-latency" default:"false" help:"Allow non-default latencies to be used in latency-based SLOs."`
 	DisableDocGeneration bool   `arg:"--no-doc,env:AM_NO_DOCGEN" default:"false" help:"Disable documentation links generation for all instrumented functions. Has the same effect as --no-doc in the //autometrics:inst directive."`
+	ProcessAllFunctions  bool   `arg:"--inst-all,env:AM_INST_ALL" default:"false" help:"Instrument all function declared in the file to transform. Overwritten by the --rm-all argument if both are set."`
```

Comment:
can we have an alias for this named `--install-all`? i know this is cool af but i see the potential for confusing typos

Reply:
I'll change the argument name sure, I didn't see the `inst-all / install` bit :D

Reply:
no dont change it its funny, just add an alias :D